authappliance/menu.py

```diff
@@ -498,7 +499,7 @@ def delete_tinc(self, vpn_name='privacyideaVPN'):
         self.display_messages()
 
     def display_messages(self):
-        self.d.scrollbox(self.info.decode('utf-8'), height=20, width=60)
+        self.d.scrollbox(self.info, height=20, width=60)
```

Comment:
Hm, redundancy setup fails at this line under Python 2 due to some encoding issues:
```
  File "/opt/venv/bin/pi-appliance", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/opt/appliance/authappliance/pi-appliance", line 23, in <module>
    main()
  File "/opt/appliance/authappliance/menu.py", line 2022, in main
    pS.main_menu()
  File "/opt/appliance/authappliance/menu.py", line 1789, in main_menu
    menu()
  File "/opt/appliance/authappliance/menu.py", line 727, in menu
    menu()
  File "/opt/appliance/authappliance/menu.py", line 803, in setup_redundancy
    self.peer.display_messages()
  File "/opt/appliance/authappliance/menu.py", line 502, in display_messages
    self.d.scrollbox(self.info, height=20, width=60)
  File "/opt/venv/local/lib/python2.7/site-packages/dialog.py", line 3412, in scrollbox
    f.write(text)
TypeError: write() argument 1 must be unicode, not str
```

... I guess we need something like privacyIDEA's ``to_unicode`` helper method here?

Reply:
I guess i have to set up some kind of redundancy to test this...

Reply:
Unfortunately i could not reproduce this error but i fixed it anyway in 8f45145

Reply:
Hm, I'm not sure why the error occurs for me :) But your commit fixes it, thanks!